### ISSUE TITLE
[Bug]: xwayland: GLX missing

### ISSUE THREAD

**Issue by dontknowhy** [termux/termux-packages#10918]
### Problem description

Xwayland doesn't have any GLX extension(i checked the build command in the package)
however,i tried the build command from tewmux-disbled(https://hub.fastgit.xyz/suhan-paradkar/tewmux-disabled)(sorry about fastgit,but that's the only way to access github…).but there is nothing work.it also broken my xfce4…

it seems like GLX is buggly.i rebuilded Xwayland,and my xfwm4 stopped working…(NEED SOME PATCHES)
i also need at least dri2 support(i can't build dri without GLX)

this is rickroll with panfrost driver…from proot…(if you can't see it,that means you can't see it)(don't ask me why does xfce works. it seems like it only used OpenGLES)

### What steps will reproduce the bug?

xfce4-session
output:(xfwm4:15761): xfwm4-WARNING **: 22:11:25.466: GLX extension missing, GLX support disabled

### What is the expected behavior?

GLX working
no libEGL:failed to create dri2 screen

### System information

termux-info:

```
Termux Variables:
TERMUX_API_VERSION=0.50.1
TERMUX_APK_RELEASE=F_DROID
TERMUX_APP_PACKAGE_MANAGER=apt
TERMUX_APP_PID=12929
TERMUX_IS_DEBUGGABLE_BUILD=0
TERMUX_MAIN_PACKAGE_FORMAT=debian
TERMUX_VERSION=0.118.0
Packages CPU architecture:
aarch64
Subscribed repositories:
# sources.list
deb https://mirrors.ustc.edu.cn/termux/apt/termux-main stable main
# sources.list.d/pointless.list
deb https://mirrors.ustc.edu.cn/termux-its-pointless/24 termux extras
# x11-repo (sources.list.d/x11.list)
deb https://mirrors.ustc.edu.cn/termux/x11-packages/ x11 main
Updatable packages:
All packages up to date
termux-tools version:
0.184
Android version:
11
Kernel build information:
Linux localhost 4.14.186+ #1 SMP PREEMPT Mon Mar 21 04:15:55 CST 2022 aarch64 Android
Device manufacturer:
HONOR
Device model:
BRT-W09

```

**Comment by dontknowhy** [comment]
I CAN USE GITHUB MOBILE FOR A DAY(or more???)
![Screenshot_20220607_223332.jpg](https://user-images.githubusercontent.com/85813508/172409121-4450102d-9103-4766-a7a7-62b2f054f273.jpg)
the rickroll you want(laugh)
https://user-images.githubusercontent.com/85813508/172409389-463d9b79-3c43-4ed0-9ce1-f0ca61c31b41.mp4

**Comment by dontknowhy** [comment]
No one answer me …
i tried to build xorg with ./autogen.sh --enable-mitshm --enable-xcsecurity --enable-xf86bigfont --enable-xwayland --enable-xorg --enable-xnest --enable-xvfb --disable-xwin --enable-xephyr --enable-kdrive --disable-devel-docs --disable-config-hal --disable-config-udev --disable-unit-tests --disable-selective-werror --disable-static --without-dtrace --disable-glamor --enable-dri --enable-dri2 --enable-dri3 --enable-glx --with-sha1=libsha1 --with-pic --prefix=$PREFIX
but it can display,but GLX still doesn't work ,and the output is the same as the termux's xwayland

**Comment by twaik** [comment]
Android will never have real GLX support because it has no OpenGL, only GLES. The only way to get it is to use mesa's OpenGL with software rendering. But it is slow...

**Comment by dontknowhy** [comment]
> Android will never have real GLX support because it has no OpenGL, only GLES. The only way to get it is to use mesa's OpenGL with software rendering. But it is slow...

yeah，
so how can i get it?

**Comment by twaik** [comment]
`pkg in mesa` ?

**Comment by dontknowhy** [comment]
> `pkg in mesa` ?

i have a container
i need to enable GLX  in the container
get it?

**Comment by dontknowhy** [comment]
> `pkg in mesa` ?

Termux:X11 with VirGL Overlay build in?

**Comment by dontknowhy** [comment]
> `pkg in mesa` is a command for termux to install package `mesa`.
> > i need to enable GLX in the container
> get it?
> 
> Check how to enable swrast in your distribution's gl implementation.
> 
> > Termux:X11 with VirGL Overlay build in?
> 
> What???
> 

EGL is not available for chromium at this time(

**Comment by xtkoba** [comment]
Already fixed by commit e43cb4947d3912f376173827e624a6f525d07641.
